interface/src/app/volcano-plot/view.js

```diff
@@ -25,13 +25,19 @@ angular.module('adage.volcano-plot.view', [
 .controller('VolcanoPlotViewCtrl', ['SampleBin', '$stateParams',
   // TODO make use of $stateParams (pulling from SampleBin for initial tests,
   //      but the right way to do this is to refactor so view.js pulls params
-  //      for sample-group-a and sample-group-b and does what's necessary to
-  //      make a plot from those lists)
+  //      for sample-base-group and sample-comp-group and does what's necessary
+  //      to make a plot from those lists)
   function VolcanoPlotViewCtrl(SampleBin, $stateParams) {
+    var ctrl = this;
+    console.log('VolcanoPlotView $stateParams:', $stateParams);
```

Comment:
Oops! Yes, that is how I debugged the problem I was having. No, I did not mean to leave it there. Thank you for catching it!